deps/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+IF (TD_ARM_32)
```

Comment:
I suggest modifying compile option in the release section only and using add_compile_options() to avoid affecting globally.